CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 [...]
 
+# v47.1.0 (07/02/2021)
```

Comment:
You forgot to update the version in the comment

Reply:
I love this file :)